asv_bench/benchmarks/utils/data_shapes.py

```diff
@@ -19,7 +19,12 @@
 from .compatibility import ASV_DATASET_SIZE, ASV_USE_STORAGE_FORMAT
 
 RAND_LOW = 0
-RAND_HIGH = 1_000_000_000 if ASV_USE_STORAGE_FORMAT == "hdk" else 100
+# use a small number of unique values in Github actions to avoid OOM (mostly related to HDK)
+RAND_HIGH = (
+    1_000_000_000
+    if ASV_USE_STORAGE_FORMAT == "hdk" and ASV_DATASET_SIZE == "Big"
```

Comment:
just to double-check, our benchmarks in dashboards already set `ASV_DATASET_SIZE=Big` so their results shouldn't change at all after this?

Reply:
yes